plugins/aws/ec2/ebsBackupEnabled.js

```diff
@@ -0,0 +1,77 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'EBS Backup Enabled',
```

Comment:
I think as this is specific to EC2 volumes, the title should be 'EBS Volume Backup Enabled'